staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -53,6 +53,33 @@ type AltNames struct {
 	IPs      []net.IP
 }
 
+// CleanAltNames cleans duplicate items in DNSNames and IPs.
+func CleanAltNames(altNames *AltNames) {
+	if altNames == nil {
+		return
+	}
+
+	dnsNamesKeys := make(map[string]struct{})
```

Comment:
FYI: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets
There are some utils you can consider to use.
```go
sets.NewString(altNames.DNSNames...).List()
```

Reply:
ok